tests/test_backends.py

```diff
@@ -451,3 +458,20 @@ def test_stac_accessor():
         "links": [],
     }
     assert stac_accessor(feat) == "S2A_11XNM_20200621_0_L2A"
+
+
+# The commented out tests work locally but fail during the build because aws creds are not configured
+@pytest.mark.parametrize(
+    "mosaic_path",
+    [
+        "file:///path/to/mosaic.json",
+        # "dynamodb://us-east-1/amosaic",
+        # "s3://mybucket/amosaic.json",
+        "https://mosaic.com/amosaic.json.gz",
+        "https://mybucket.s3.amazonaws.com/mosaic.json",
+    ],
+)
```

Comment:
yeah maybe we can mock boto3 response? it's more work so I'm 👌 merging this for now